example/index.js

```diff
@@ -1,19 +1,67 @@
 const Steveo = require('steveo').default;
 const { kafkaCompression } = require('steveo');
 
-const env = Object.assign({}, process.env, { KAFKA_CODEC: kafkaCompression.GZIP });
-
-const steveo = Steveo(env, console);
+const publishCallback = {
+  success: (topic, payload) => console.log(topic, payload, 'Yaaay success'),
+  failure: (topic, payload) => console.log(topic, payload, 'Ohhh failed'),
+};
 
-// wait for the kafka consumer to call the subscribe action
-const subscribe = async (payload) => {
-  console.log('Here is the payload', payload);
+const config = {
+  kafkaConnection: process.env.KAFKA_CONNECTION,
+  kafkaCodec: kafkaCompression.GZIP,
+  clientId: '1234-123',
+  logLevel: 5,
+  kafkaGroupId: '1234',
+  kafkaSendAttempts: process.env.KAFKA_SEND_ATTEMPTS,
+  kafkaSendDelayMin: process.env.KAFKA_SEND_DELAY_MIN,
+  kafkaSendDelayMax: process.env.KAFKA_SEND_DELAY_MAX,
+  publishCallback,
 };
 
-// define task
-steveo.task.define('test-topic', subscribe);
+(async () => {
+  const steveo = Steveo(config, console);
+
+  // subscribe Call for first task
+  const subscribe = async (payload) => {
+    console.log('Payload from first producer', payload);
+  };
+
+  // create first Task
+  const firstTask = steveo.task();
+
+  // define Task
+  await firstTask.define('test-topic', subscribe);
```

Comment:
think about our use case - we define 30 tasks. if they all have `await` at boot time?